qiskit_experiments/data_processing/data_action.py

```diff
@@ -81,16 +102,42 @@ def __repr__(self):
 class TrainableDataAction(DataAction):
     """A base class for data actions that need training."""
 
+    def __init__(self, validate: bool = True):
+        """Create new node.
+
+        Args:
+            validate: If set to False the DataAction will not validate its input.
+        """
+        super().__init__(validate=validate)
+        self._parameters = self._default_parameters()
+
+    @classmethod
+    def _default_parameters(cls) -> Options:
+        """Set default node parameters."""
```

Comment:
Please add more documentation explaining why we need this and how it is relevant for sub-classes.

Reply:
added in https://github.com/Qiskit/qiskit-experiments/pull/639/commits/1c0b99d7f26afbe79bec1e1cb2d0d19320087dc0